CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Release Notes
 
+### 0.11.3
+
+This patch release:
+
+* Adds handling for common JSON parsing exceptions and wraps them in a `JwtException`.
```

Comment:
A minor nit: to avoid a suspicion that that we didn't handle JSON parsing exceptions previously (we did and do), I'd write this as:

> Adds additional handling for rare JSON parsing exceptions and wraps them in a \`JwtException\` to allow the application to handle these conditions as JWT concerns.

Reply:
I like it!! Thanks!